File: employee-app/employee-service/src/main/java/com/p8/io/employee/service/model/Employee.java
package com.p8.io.employee.service.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.*;

@Entity
@Table(name="employee")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Data
@Builder
public class Employee {

    @Id
    private String id;

    @Column(name = "first_name")
    private String firstName;

    @Column(name = "last_name")
    private String lastName;

    @Column(name = "email_id")
    private String emailId;

}

File: employee-app/employee-react-ui/src/interfaces/Employee.ts
export interface IEmployee {
    id?: string | null,
    firstName: string,
    lastName: string,
    emailId: string,
}

export class Employee implements IEmployee {
    public id: null; 
    public firstName: string;
    public lastName: string;
    public emailId: string; 
 
    constructor(){
        this.id = null; 
        this.firstName = "";
        this.lastName = "";
        this.emailId = ""; 
    }
}